docker/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/community" >>/etc/apk/reposi
 	apk add --update bash runit conntrack-tools iproute2 util-linux curl && \
 	rm -rf /var/cache/apk/*
 ADD ./docker.tgz /
+ADD ./ebpf.tgz /usr/libexec/scope/
```

Comment:
It is only one file now (ebpf.o). It used to be one file per kernel version, but not anymore.

If we release the file as part of releases of tcptracer-bpf on github, is there a way to upload that file automatically after a build in the CI?